### PR TITLE
[doc] Added some words about unsupported processes in "Create a simple parallel model"

### LINKED ISSUE #2455: running THM benchmarks lead to PETSC "Argument out of range" error
### Description

Running any of the THM benchmarks on my computer lead to a petsc "argument out of range" error.
Even deactivating PETSC in the input script results in the usage of PETSC and produces the same error:

>

>  [0]PETSC ERROR: --------------------- Error Message --------------------------------------------------------------
> [0]PETSC ERROR: Argument out of range
> [0]PETSC ERROR: Out of range index value 8094 maximum 8092
> [0]PETSC ERROR: See http://www.mcs.anl.gov/petsc/documentation/faq.html for trouble shooting.
> [0]PETSC ERROR: Petsc Release Version 3.8.3, Dec, 09, 2017 
> [0]PETSC ERROR: ogs on a arch-linux2-c-debug named modmon041 by buchwalj Mon Apr  8 11:55:06 2019
> [0]PETSC ERROR: Configure options --with-fc=0 --download-f2cblaslapack=1 --prefix=../install
> [0]PETSC ERROR: #1 VecSetValues_MPI() line 950 in /home/buchwalj/.conan/data/petsc/3.8.3/bilke/testing/build/b80d46004713aa37d6a90b42e2a326a056a237b5/source/src/vec/vec/impls/mpi/pdvec.c
> [0]PETSC ERROR: #2 VecSetValues() line 854 in /home/buchwalj/.conan/data/petsc/3.8.3/bilke/testing/build/b80d46004713aa37d6a90b42e2a326a056a237b5/source/src/vec/vec/interface/rvector.c
> [0]PETSC ERROR: --------------------- Error Message --------------------------------------------------------------
> [0]PETSC ERROR: Argument out of range
> [0]PETSC ERROR: Out of range index value 8098 maximum 8092
> [0]PETSC ERROR: See http://www.mcs.anl.gov/petsc/documentation/faq.html for trouble shooting.
> [0]PETSC ERROR: Petsc Release Version 3.8.3, Dec, 09, 2017 
> [0]PETSC ERROR: ogs on a arch-linux2-c-debug named modmon041 by buchwalj Mon Apr  8 11:55:06 2019
> [0]PETSC ERROR: Configure options --with-fc=0 --download-f2cblaslapack=1 --prefix=../install
> [0]PETSC ERROR: #3 VecSetValues_MPI() line 950 in /home/buchwalj/.conan/data/petsc/3.8.3/bilke/testing/build/b80d46004713aa37d6a90b42e2a326a056a237b5/source/src/vec/vec/impls/mpi/pdvec.c
> [0]PETSC ERROR: #4 VecSetValues() line 854 in /home/buchwalj/.conan/data/petsc/3.8.3/bilke/testing/build/b80d46004713aa37d6a90b42e2a326a056a237b5/source/src/vec/vec/interface/rvector.c
> [0]PETSC ERROR: --------------------- Error Message --------------------------------------------------------------
> [0]PETSC ERROR: Argument out of range
> [0]PETSC ERROR: Out of range index value 8102 maximum 8092
> [0]PETSC ERROR: See http://www.mcs.anl.gov/petsc/documentation/faq.html for trouble shooting.
> [0]PETSC ERROR: Petsc Release Version 3.8.3, Dec, 09, 2017 
> [0]PETSC ERROR: ogs on a arch-linux2-c-debug named modmon041 by buchwalj Mon Apr  8 11:55:06 2019
> [0]PETSC ERROR: Configure options --with-fc=0 --download-f2cblaslapack=1 --prefix=../install
> [0]PETSC ERROR: #5 VecSetValues_MPI() line 950 in /home/buchwalj/.conan/data/petsc/3.8.3/bilke/testing/build/b80d46004713aa37d6a90b42e2a326a056a237b5/source/src/vec/vec/impls/mpi/pdvec.c
> [0]PETSC ERROR: #6 VecSetValues() line 854 in /home/buchwalj/.conan/data/petsc/3.8.3/bilke/testing/build/b80d46004713aa37d6a90b42e2a326a056a237b5/source/src/vec/vec/interface/rvector.c
> [0]PETSC ERROR: --------------------- Error Message --------------------------------------------------------------
> [0]PETSC ERROR: Argument out of range
> [0]PETSC ERROR: Out of range index value 8106 maximum 8092
> [0]PETSC ERROR: See http://www.mcs.anl.gov/petsc/documentation/faq.html for trouble shooting.
> [0]PETSC ERROR: Petsc Release Version 3.8.3, Dec, 09, 2017 
> [0]PETSC ERROR: ogs on a arch-linux2-c-debug named modmon041 by buchwalj Mon Apr  8 11:55:06 2019
> [0]PETSC ERROR: Configure options --with-fc=0 --download-f2cblaslapack=1 --prefix=../install
> [0]PETSC ERROR: #7 VecSetValues_MPI() line 950 in /home/buchwalj/.conan/data/petsc/3.8.3/bilke/testing/build/b80d46004713aa37d6a90b42e2a326a056a237b5/source/src/vec/vec/impls/mpi/pdvec.c
> [0]PETSC ERROR: #8 VecSetValues() line 854 in /home/buchwalj/.conan/data/petsc/3.8.3/bilke/testing/build/b80d46004713aa37d6a90b42e2a326a056a237b5/source/src/vec/vec/interface/rvector.c
> [0]PETSC ERROR: --------------------- Error Message --------------------------------------------------------------
> [0]PETSC ERROR: Argument out of range
> [0]PETSC ERROR: Out of range index value 8110 maximum 8092
> [0]PETSC ERROR: See http://www.mcs.anl.gov/petsc/documentation/faq.html for trouble shooting.
> [0]PETSC ERROR: Petsc Release Version 3.8.3, Dec, 09, 2017 
> [0]PETSC ERROR: ogs on a arch-linux2-c-debug named modmon041 by buchwalj Mon Apr  8 11:55:06 2019
> [0]PETSC ERROR: Configure options --with-fc=0 --download-f2cblaslapack=1 --prefix=../install
> [0]PETSC ERROR: #9 VecSetValues_MPI() line 950 in /home/buchwalj/.conan/data/petsc/3.8.3/bilke/testing/build/b80d46004713aa37d6a90b42e2a326a056a237b5/source/src/vec/vec/impls/mpi/pdvec.c
> [0]PETSC ERROR: #10 VecSetValues() line 854 in /home/buchwalj/.conan/data/petsc/3.8.3/bilke/testing/build/b80d46004713aa37d6a90b42e2a326a056a237b5/source/src/vec/vec/interface/rvector.c
> [0]PETSC ERROR: --------------------- Error Message --------------------------------------------------------------
> [0]PETSC ERROR: Argument out of range
> [0]PETSC ERROR: Out of range index value 8114 maximum 8092
> [0]PETSC ERROR: See http://www.mcs.anl.gov/petsc/documentation/faq.html for trouble shooting.
> [0]PETSC ERROR: Petsc Release Version 3.8.3, Dec, 09, 2017 
> [0]PETSC ERROR: ogs on a arch-linux2-c-debug named modmon041 by buchwalj Mon Apr  8 11:55:06 2019
> [0]PETSC ERROR: Configure options --with-fc=0 --download-f2cblaslapack=1 --prefix=../install
> [0]PETSC ERROR: #11 VecSetValues_MPI() line 950 in /home/buchwalj/.conan/data/petsc/3.8.3/bilke/testing/build/b80d46004713aa37d6a90b42e2a326a056a237b5/source/src/vec/vec/impls/mpi/pdvec.c
> [0]PETSC ERROR: #12 VecSetValues() line 854 in /home/buchwalj/.conan/data/petsc/3.8.3/bilke/testing/build/b80d46004713aa37d6a90b42e2a326a056a237b5/source/src/vec/vec/interface/rvector.c
> [0]PETSC ERROR: --------------------- Error Message --------------------------------------------------------------
[... ]

> [0]PETSC ERROR: --------------------- Error Message --------------------------------------------------------------
> [0]PETSC ERROR: Argument out of range
> [0]PETSC ERROR: Column too large: col 12022 max 8091
> [0]PETSC ERROR: See http://www.mcs.anl.gov/petsc/documentation/faq.html for trouble shooting.
> [0]PETSC ERROR: Petsc Release Version 3.8.3, Dec, 09, 2017 
> [0]PETSC ERROR: ogs on a arch-linux2-c-debug named modmon041 by buchwalj Mon Apr  8 11:54:42 2019
> [0]PETSC ERROR: Configure options --with-fc=0 --download-f2cblaslapack=1 --prefix=../install
> [0]PETSC ERROR: #6571 MatSetValues_MPIAIJ() line 579 in /home/buchwalj/.conan/data/petsc/3.8.3/bilke/testing/build/b80d46004713aa37d6a90b42e2a326a056a237b5/source/src/mat/impls/aij/mpi/mpiaij.c
> [0]PETSC ERROR: #6572 MatSetValues() line 1299 in /home/buchwalj/.conan/data/petsc/3.8.3/bilke/testing/build/b80d46004713aa37d6a90b42e2a326a056a237b5/source/src/mat/interface/matrix.c
> [0] [time] Assembly took 0.149516 s.
> [0]PETSC ERROR: --------------------- Error Message --------------------------------------------------------------
> [0]PETSC ERROR: Object is in wrong state
> [0]PETSC ERROR: Matrix is missing diagonal entry in the zeroed row 836
> [0]PETSC ERROR: See http://www.mcs.anl.gov/petsc/documentation/faq.html for trouble shooting.
> [0]PETSC ERROR: Petsc Release Version 3.8.3, Dec, 09, 2017 
> [0]PETSC ERROR: ogs on a arch-linux2-c-debug named modmon041 by buchwalj Mon Apr  8 11:54:42 2019
> [0]PETSC ERROR: Configure options --with-fc=0 --download-f2cblaslapack=1 --prefix=../install
> [0]PETSC ERROR: #6573 MatZeroRows_SeqAIJ() line 1824 in /home/buchwalj/.conan/data/petsc/3.8.3/bilke/testing/build/b80d46004713aa37d6a90b42e2a326a056a237b5/source/src/mat/impls/aij/seq/aij.c
> [0]PETSC ERROR: #6574 MatZeroRows() line 5774 in /home/buchwalj/.conan/data/petsc/3.8.3/bilke/testing/build/b80d46004713aa37d6a90b42e2a326a056a237b5/source/src/mat/interface/matrix.c
> [0]PETSC ERROR: #6575 MatZeroRows_MPIAIJ() line 797 in /home/buchwalj/.conan/data/petsc/3.8.3/bilke/testing/build/b80d46004713aa37d6a90b42e2a326a056a237b5/source/src/mat/impls/aij/mpi/mpiaij.c
> [0]PETSC ERROR: #6576 MatZeroRows() line 5774 in /home/buchwalj/.conan/data/petsc/3.8.3/bilke/testing/build/b80d46004713aa37d6a90b42e2a326a056a237b5/source/src/mat/interface/matrix.c
> [0] [time] Applying Dirichlet BCs took 0.00125145 s.
> [0] Convergence criterion: |r0|=9.2873e+06
> [0]PETSC ERROR: --------------------- Error Message --------------------------------------------------------------
> [0]PETSC ERROR: Object is in wrong state
> [0]PETSC ERROR: Matrix is missing diagonal entry 228
> [0]PETSC ERROR: See http://www.mcs.anl.gov/petsc/documentation/faq.html for trouble shooting.
> [0]PETSC ERROR: Petsc Release Version 3.8.3, Dec, 09, 2017 
> [0]PETSC ERROR: ogs on a arch-linux2-c-debug named modmon041 by buchwalj Mon Apr  8 11:54:42 2019
> [0]PETSC ERROR: Configure options --with-fc=0 --download-f2cblaslapack=1 --prefix=../install
> [0]PETSC ERROR: #6577 MatILUFactorSymbolic_SeqAIJ() line 1687 in /home/buchwalj/.conan/data/petsc/3.8.3/bilke/testing/build/b80d46004713aa37d6a90b42e2a326a056a237b5/source/src/mat/impls/aij/seq/aijfact.c
> [0]PETSC ERROR: #6578 MatILUFactorSymbolic() line 6610 in /home/buchwalj/.conan/data/petsc/3.8.3/bilke/testing/build/b80d46004713aa37d6a90b42e2a326a056a237b5/source/src/mat/interface/matrix.c
> [0]PETSC ERROR: #6579 PCSetUp_ILU() line 144 in /home/buchwalj/.conan/data/petsc/3.8.3/bilke/testing/build/b80d46004713aa37d6a90b42e2a326a056a237b5/source/src/ksp/pc/impls/factor/ilu/ilu.c
> [0]PETSC ERROR: #6580 PCSetUp() line 924 in /home/buchwalj/.conan/data/petsc/3.8.3/bilke/testing/build/b80d46004713aa37d6a90b42e2a326a056a237b5/source/src/ksp/pc/interface/precon.c
> [0]PETSC ERROR: #6581 KSPSetUp() line 381 in /home/buchwalj/.conan/data/petsc/3.8.3/bilke/testing/build/b80d46004713aa37d6a90b42e2a326a056a237b5/source/src/ksp/ksp/interface/itfunc.c
> [0]PETSC ERROR: #6582 PCSetUpOnBlocks_BJacobi_Singleblock() line 618 in /home/buchwalj/.conan/data/petsc/3.8.3/bilke/testing/build/b80d46004713aa37d6a90b42e2a326a056a237b5/source/src/ksp/pc/impls/bjacobi/bjacobi.c
> [0]PETSC ERROR: #6583 PCSetUpOnBlocks() line 955 in /home/buchwalj/.conan/data/petsc/3.8.3/bilke/testing/build/b80d46004713aa37d6a90b42e2a326a056a237b5/source/src/ksp/pc/interface/precon.c
> [0]PETSC ERROR: #6584 KSPSetUpOnBlocks() line 213 in /home/buchwalj/.conan/data/petsc/3.8.3/bilke/testing/build/b80d46004713aa37d6a90b42e2a326a056a237b5/source/src/ksp/ksp/interface/itfunc.c
> [0]PETSC ERROR: #6585 KSPSolve() line 613 in /home/buchwalj/.conan/data/petsc/3.8.3/bilke/testing/build/b80d46004713aa37d6a90b42e2a326a056a237b5/source/src/ksp/ksp/interface/itfunc.c
> 
> Divergence detected, use command option -ksp_monitor or -log_summary to check the details.
> [0] [time] Linear solver took 0.00169956 s.
> [0] Newton: The linear solver failed.
> [0] [time] Solving process #0 took 0.154208 s in time step #1 
> [0] The nonlinear solver failed in time step #1 at t = 100 s for process #0.
> [0] [time] Output of timestep 1 took 0.0155461 s.
> [0] Nonlinear solver fails. Because the time stepper FixedTimeStepping is used, the program has to be terminated. at UncoupledProcessesTimeLoop.cpp, line 651
> [0] OGS terminated on 2019-04-08 11:54:42+020
> [0] OGS terminated with error.

## Specifications

  - Version:  6.2.0-44-gc93535842
  - Platform: 4.19.0-4-amd64 #1 SMP Debian 4.19.28-2 (2019-03-15) x86_64 GNU/Linux

### PR DESCRIPTION
**Pull request by wenqing**
As titled.

Closes #2455.

### CODE REVIEW

File: web/content/docs/tools/workflows/create-a-simple-parallel-model/index.pandoc [hunk 1]
```diff
@@ -8,6 +8,12 @@ author = "Thomas Fischer"
     parent = "workflows"
 +++
 
+## Unsupported Processes
+So far, HydroMechanics (including that with LIE) and ThermoHydroMechanics, which
+ use linear element for thermal and hydraulic processes and quadratic element
```
Comment (bilke) [inline web/content/docs/tools/workflows/create-a-simple-parallel-model/index.pandoc]:
```suggestion
 use linear elements for thermal and hydraulic processes and quadratic elements
```

File: web/content/docs/tools/workflows/create-a-simple-parallel-model/index.pandoc [hunk 2]
```diff
@@ -8,6 +8,12 @@ author = "Thomas Fischer"
...
+ for mechanics process, respectively, can not use the parallel model.
```
Comment (bilke) [inline web/content/docs/tools/workflows/create-a-simple-parallel-model/index.pandoc]:
```suggestion
 for the mechanics process, respectively, can not be used in the parallel model.
```

File: web/content/docs/tools/workflows/create-a-simple-parallel-model/index.pandoc (hunk 1 above)
Comment (wenqing) [reply web/content/docs/tools/workflows/create-a-simple-parallel-model/index.pandoc]: Thanks. I just changed the way of the expression. Please take  a look again.

File: web/content/docs/tools/workflows/create-a-simple-parallel-model/index.pandoc [hunk 3]
```diff
@@ -8,6 +8,16 @@ author = "Thomas Fischer"
     parent = "workflows"
 +++
 
+## Unsupported Processes
+So far, the parallel FEM scheme with PETSc cannot be used by the following processes:
```
Comment (endJunction) [inline web/content/docs/tools/workflows/create-a-simple-parallel-model/index.pandoc]:
```suggestion
So far, the parallel FEM scheme with PETSc cannot be used by all processes using linear and quadratic function spaces simultaniously as, for example:
```

File: web/content/docs/tools/workflows/create-a-simple-parallel-model/index.pandoc [hunk 4]
```diff
@@ -8,6 +8,16 @@ author = "Thomas Fischer"
...
+
+- HydroMechanics (including that with LIE)
+- ThermoHydroMechanics
```
Comment (endJunction) [inline web/content/docs/tools/workflows/create-a-simple-parallel-model/index.pandoc]:
```suggestion
- ThermoMechanics
- ThermoHydroMechanics
- RichardsMechanics
```

Comment (wenqing) [reply web/content/docs/tools/workflows/create-a-simple-parallel-model/index.pandoc]: @endJunction Thanks. ThermoMechanics is "luck" in this issue, it uses the same order of elements for both T and M. 

Took the other suggestions by amending the commit.

File: web/content/docs/tools/workflows/create-a-simple-parallel-model/index.pandoc [hunk 5]
```diff
@@ -8,6 +8,14 @@ author = "Thomas Fischer"
     parent = "workflows"
 +++
 
+## Unsupported Processes
+So far, the parallel FEM scheme with PETSc cannot be used by the all processes
+ using linear and quadratic function spaces simultaneously as. They are
```
Comment (endJunction) [inline web/content/docs/tools/workflows/create-a-simple-parallel-model/index.pandoc]: Sorry, Wenqing, there is one more mistake in the sentence: it ends with `as.`, which is probably wrong. Merge after fixing it though!

Comment (wenqing) [reply web/content/docs/tools/workflows/create-a-simple-parallel-model/index.pandoc]: oops!

**Pull request merged**